bundles/best_practices.rst

```diff
@@ -171,73 +171,59 @@ Continuous Integration
-
-Here is the recommended configuration file (``.travis.yml``) for Symfony bundles,
-which test the two latest :doc:`LTS versions </contributing/community/releases>`
-of Symfony and the latest beta release:
-
-.. code-block:: yaml
-
-    language: php
-
-    cache:
-        directories:
-            - $HOME/.composer/cache/files
-            - $HOME/symfony-bridge/.phpunit
-
-    env:
-        global:
-            - PHPUNIT_FLAGS="-v"
-            - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
-
-    matrix:
-        fast_finish: true
-        include:
-              # Minimum supported dependencies with the latest and oldest PHP version
-            - php: 7.2
-              env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="max[self]=0"
-            - php: 7.1
-              env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="max[self]=0"
-
-              # Test the latest stable release
-            - php: 7.1
-            - php: 7.2
-              env: COVERAGE=true PHPUNIT_FLAGS="-v --coverage-text"
-
-              # Test LTS versions. This makes sure we do not use Symfony packages with version greater
-              # than 2 or 3 respectively. Read more at https://github.com/symfony/lts
-            - php: 7.2
-              env: DEPENDENCIES="symfony/lts:^2"
-            - php: 7.2
-              env: DEPENDENCIES="symfony/lts:^3"
-
-              # Latest commit to master
-            - php: 7.2
-              env: STABILITY="dev"
-
-        allow_failures:
-              # Dev-master is allowed to fail.
-            - env: STABILITY="dev"
-
-    before_install:
-        - if [[ $COVERAGE != true ]]; then phpenv config-rm xdebug.ini || true; fi
-        - if ! [ -z "$STABILITY" ]; then composer config minimum-stability ${STABILITY}; fi;
-        - if ! [ -v "$DEPENDENCIES" ]; then composer require --no-update ${DEPENDENCIES}; fi;
-
-    install:
-        - composer update ${COMPOSER_FLAGS} --prefer-dist --no-interaction
-        - ./vendor/bin/simple-phpunit install
-
-    script:
-        - composer validate --strict --no-check-lock
-        # simple-phpunit is the PHPUnit wrapper provided by the PHPUnit Bridge component and
-        # it helps with testing legacy code and deprecations (composer require symfony/phpunit-bridge)
-        - ./vendor/bin/simple-phpunit $PHPUNIT_FLAGS
-
-Consider using the `Travis cron`_ tool to make sure your project is built even if
-there are no new pull requests or commits.
+providing this feature for free for open source projects, like `GitHub Actions`_
+and `Travis CI`_.
+
+A bundle should at least test:
+
+* The lower bound of their dependencies (by running ``composer update --prefer-lowest``);
+* The supported PHP versions;
+* All supported major Symfony versions (e.g. both ``3.x`` and ``4.x`` if
+  support is claimed for both).
+
+Thus, a bundle support PHP 7.3, 7.4 and 8.0, and Symfony 3.4 and 4.x should
```

Comment:
Is it still worth documenting v3.4?

Also for the travis config file, is it possible to use the langage toggle config feature php/xml/yaml but instead use it for toggling between for example github actions, travis, etc ?

Reply:
```suggestion
Thus, a bundle supporting PHP 7.3, 7.4 and 8.0, and Symfony 3.4, 4.x and 5.x should
```
listing explicit versions is prone to be outdated after a while, but i think this is about the example, so getting outdated is ok.